logbook-json/src/main/java/org/zalando/logbook/json/JacksonJsonFieldBodyFilter.java

```diff
@@ -0,0 +1,80 @@
+package org.zalando.logbook.json;
+
+import java.io.StringWriter;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+import org.zalando.logbook.BodyFilter;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * 
+ * Thread-safe filter for JSON fields. Filters on property names, 
```

Comment:
Can we document that the side effect of this filter is compacting of the JSON body and therefore shouldn't be used in conjunction with the compacting body filter.

Reply:
Ok, I guess then adding compacting logic on the exception handling makes sense.

Reply:
So I'll wait for https://github.com/zalando/logbook/pull/528/files

Reply:
Added compacting filter on exception